DCC.combined-schema.json

```diff
@@ -111,7 +120,7 @@
           "type": "string",
           "maxLength": 80,
           "examples": [
-            "d'Červenková Panklová"
+            "d'\u00c4\u0152ervenkov\u00c3\u00a1 Panklov\u00c3\u00a1"
```

Comment:
Technically identical, but it's weird that this is now escaped like this, instead of properly UTF-8-encoded. (Same for line 142.)

Reply:
Arggghhh no that's not good. F****** Windows.

Reply:
Fixed it thanks to some stackoverflow posts. The Python documentation was again rather useless.